README.md

```diff
@@ -87,9 +87,8 @@ need them.
 
 ### Platform support
 
-While Rustls itself is platform independent, by default it uses
-[`ring`](https://crates.io/crates/ring) for implementing the cryptography in
-TLS. As a result, rustls only runs on platforms
+While Rustls itself is platform independent, by default it uses [`ring`] for implementing
+the cryptography in TLS. As a result, rustls only runs on platforms
 supported by `ring`. At the time of writing, this means 32-bit ARM, Aarch64 (64-bit ARM),
 x86, x86-64, LoongArch64, 32-bit & 64-bit Little Endian MIPS, 32-bit PowerPC (Big Endian),
 64-bit PowerPC (Big and Little Endian), 64-bit RISC-V, and s390x. We do not presently
```

Comment:
It would be better if you didn't list all these platforms because they will get out of date, and it will also be extra work to maintain this list as potentially every time we add a target architecture, we have to update this list. There is such a list where BoringSSL or AWS-LC or medTLS are mentioned.

Also, "platform" is mostly a synonym for "target" and the OS (or lack of an OS) is another limiting factor.

Reply:
Especially now that rustls abstracts over crypto provider, I agree that it probably doesn't make sense to list platforms/targets here explicitly.

Reply:
There have been a few different iterations on the best way to describe platform support. @briansmith you approved the PR that last adjusted this text: https://github.com/rustls/rustls/pull/1532

Since this text is pre-existing and unmodified by my changes .I would prefer to limit the scope of this branch to how it's described: adding more documentation on the crypto providers that we know exist. 

It seems like there's a variety of opinions on how best to describe the platform support so I think it merits its own PR and discussion.